cmd/ethrex/cli.rs

```diff
@@ -135,6 +135,32 @@ pub struct Options {
         env = "ETHREX_HTTP_PORT"
     )]
     pub http_port: String,
+    #[arg(
+        long = "ws",
```

Comment:
To be consistent with `p2p.enabled`
```suggestion
        long = "ws.enabled",
```